apps/dashboard/src/app/nebula-app/(app)/components/CustomChat/CustomChatButton.tsx

```
@@ -0,0 +1,93 @@
+import { Button } from "@/components/ui/button";
+import { XIcon, MessageCircleIcon } from "lucide-react";
+import { cn } from "@/lib/utils";
+
+export function CustomChatButton(props: {
+  isLoggedIn: boolean;
+  networks: "mainnet" | "testnet" | "all" | null;
+  isFloating: boolean;
+  pageType: "chain" | "contract" | "support";
+  label: string;
+  client: ThirdwebClient;
+  customApiParams: any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)